metrics/deploy/prometheus-ocs-rules.yaml

```diff
@@ -449,3 +449,15 @@ spec:
       for: 5m
       labels:
         severity: warning
+    - alert: DuplicateStorageAutoScalerCRsFound
+      annotations:
+        description: |-
+          Duplicate StorageAutoScaler CRs found in the namespace {{ $labels.namespace }} trying to scale device class {{ $labels.device_class }} under storage cluster {{ $labels.storagecluster }}.
+        message: Duplicate StorageAutoScaler CRs found.
+        runbook_url: https://github.com/openshift/runbooks/blob/master/alerts/openshift-container-storage-operator/DuplicateStorageAutoScalerCRsFound.md
+        severity_level: warning
+      expr: |
+        count by (namespace, managedBy, storagecluster, device_class) (ocs_storageautoscaler_capacity_reached) > 1
```

Comment:
I thought we would be creating separate metrics for duplicate CR by a new `duplicate: true` status spec field, @umangachapagain, what are your thoughts?

Reply:
IMHO, we don't really require a separate backend implementation (like this [PR#3130](https://github.com/red-hat-storage/ocs-operator/pull/3130)). This is a common thing to do through prometheus query.

If we are going with the [PR#3130](https://github.com/red-hat-storage/ocs-operator/pull/3130) backend approach, we really don't require a separate alert. [When we detect duplicate CRs in the backend] We just have to update the CR with a failure status and an alert will be automatically fired (as we have alert for the failure cases).

Reply:
Okay lets go with the current approach

Reply:
This approach is better.
If backend sets the error phase, we'll probably have 2 alerts. Do we want that?
Or do you want to handle scaling error vs duplicate CRs in runbook?

Reply:
From the backend we wont set it as failed state, but will introduce a new state as Invalid